precovery/frame_db.py

```diff
@@ -393,27 +385,41 @@ def all_frames(self) -> Iterator[HealpixFrame]:
         for row in result:
             yield HealpixFrame(*row)
 
-    def add_frame(self, frame: HealpixFrame):
-        insert = self.frames.insert().values(
-            dataset_id=frame.dataset_id,
-            obscode=frame.obscode,
-            exposure_id=frame.exposure_id,
-            filter=frame.filter,
-            mjd=frame.mjd,
-            healpixel=int(frame.healpixel),
-            data_uri=frame.data_uri,
-            data_offset=frame.data_offset,
-            data_length=frame.data_length,
-        )
-        self.dbconn.execute(insert)
+    def add_frames(self, frames: list[HealpixFrame]):
```

Comment:
Probably should be `frames: Iterable[HealpixFrame]`?

Reply:
Ah, I reveal that my `typing` knowledge is from before Python 3.9: https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections